.github/workflows/tests-integration-db.yml

```diff
@@ -44,13 +44,7 @@ jobs:
       - name: Poetry install
         run: poetry run -- nox -s run_in_dev_env -- poetry install
 
-      - name: Build language container
-        run: poetry run -- nox -s build_language_container
-
-      - name: Start test environment
-        run: poetry run -- nox -s start_integration_test_environment
-
       - name: Run Python integration tests with db
-        run: poetry run -- nox -s run_python_integration_tests_with_db
+        run: poetry run -- nox -s run_python_integration_tests_with_db -- -- --setup-show --backend=onprem
```

Comment:
This PR excludes Exasol SaaS backend.

Supporting SaaS requires switching to new BFSP API, which will be addressed in the scope of the separate ticket #176. According to @tkilias SaaS support in general for AAF does not have high priority and can wait until next year.